bindings/jupyter/js/src/widgets/index.js

```diff
@@ -1,5 +1,15 @@
-export * from './StackedCalendarWidget';
-export * from './GraphBuilderWidget';
-export * from './FeatureListViewWidget';
-export * from './MultiWayPlotWidget';
-export * from './ManifoldWidget';
+const widgetBuilder = require('./react-widget-builder').default;
+
+for (const name of [
+  'StackedCalendar',
+  'GraphBuilder',
+  'FeatureListView',
+  'MultiWayPlot',
+  'Manifold',
+]) {
+  const Component = require('../components')[name];
```

Comment:
Shall we add sanity checks and throw warnings when `Component` does not exist, e.g. due to typos in the Component names above.

Reply:
In fact, if the component name is incorrect, the require statement will throw an error and the error message is usually good enough to help figure out what happens.

Reply:
Added a TODO above to indicate this piece of code is not going to stay permanent.

```javascript
// TODO: Implement dynamic import of components
for (const name of [
  'StackedCalendar',
  'GraphBuilder',
  'FeatureListView',
  'MultiWayPlot',
  'Manifold',
]) {
```